src/components/imageCarousel/ImageSlider.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { ImageSlider } from './ImageSlider';
+
+export default {
+  title: 'Components/ImageSlider',
+  component: ImageSlider,
+  args: {
+    data: [
+      {
+        title: 'Hello world',
+        subtitle: 'Thank you',
+        img: 'https://s3-us-east-2.amazonaws.com/maryville/wp-content/uploads/2020/12/16162130/computer-programmer.jpg',
+      },
+      {
+        title: 'Amazing!',
+        subtitle: 'All of our savings are gone',
+        img: 'https://www.beedigital.es/wp-content/uploads/2020/09/342c4d9715440a942a78e4417fbcbad2155746109-1.jpg',
+      },
+      {
+        title: 'Erik (19)',
+        subtitle: 'Programming is fun and not stressful',
```

Comment:
this is not related to this ticket

Reply:
It wasnt part of the intended pr, just a residue of the previous branch.
Let me delete this.